source/_integrations/androidtv_remote.markdown

```diff
@@ -283,6 +283,29 @@ target:
   entity_id: remote.living_room_tv
 ```
 
+### `androidtv_remote.send_text`
+
+The `androidtv_remote.send_text` action allows you to send text to your Android TV device.
```

Comment:
```suggestion
The `androidtv_remote.send_text` action allows you to send text as input to your Android TV device. This allows you to fill text fields as if it was typed using the keyboard.
```

When I was reading this I was wondering where would the text show up, and if was some kind of notification, until I realized that it was for "typing", so I think it would be nice to have a clearer description (my suggestion is just a suggestion, feel free to reword and adapt)